pages/index.js

```diff
@@ -87,225 +87,215 @@ class IndexPage extends Component {
 
   render () {
     return (
-      <PageWrapper title='The Teacher Fund'>
-        <div className='bg-white index tf-dark-gray'>
-          <section className='index__header h-section pv7 ph2 pr7-l cover-l'>
-            <div className='fr flex flex-column mt5 mt0-l w-90-m'>
-              <div className='white f2 tf-oswald fl ml-4-l f-5-l'>
-                Funding Teachers.<br />
-                Empowering Students.
-              </div>
-              <div className='ts-subtext fl tf-lato-lite white'>
-                <p>We provide funding to classrooms so teachers don't have to.</p>
-              </div>
-            </div>
-          </section>
-          <section className='h-section ph2 pt4 pt6-l'>
-            <div className='tf-lato-lite ts-subtext w-auto m-auto center-l tc ph3 ph5-m pb4-m ph0-l w-50-l'>
-              <p className='lh-copy ma0 f4 f3-ns mb4'>
+      <div className='bg-white index tf-dark-gray'>
+        <PageWrapper title='The Teacher Fund' />
```

Comment:
@joelwass Yes, the changes break the accessibility that the skip link offers (which is helpful for screenreader users as well as those navigating with their keyboard, to avoid tabbing through the logo and nav on multiple pages).

The page wrapper component wraps the main content in a `main` element which the skip link puts into focus when clicked, but with the changes, the `main` element is empty, so nothing happens when the skip link is clicked. The rendered markup gives you a bit of an idea of the problem:
![image](https://user-images.githubusercontent.com/16426195/75179642-22949200-5700-11ea-9f51-19fd3f8aac5d.png)

Reply:
Awesome thanks @meowwwls! 

Does it change much to put the Page wrapper back where it was? @JennaKoslowski

Reply:
@joelwass @JennaKoslowski 

Just taking a quick look, it doesn't look like it should be an issue to have it how it was before the changes, simply replacing the header content with the `Hero` component. I could be overlooking something, though!

Reply:
Changing the Page Wrapper back to how it was doesn't change anything. I edited it and pushed it.